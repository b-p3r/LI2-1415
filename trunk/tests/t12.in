9 9
SCLAEDEEM
NAANMUROA
ORTICISMI
IEOAOVOTO
OMSAGDOFA
ISMOAUARA
TECASLORI
LOECIMDME
FEGOEEMAS
