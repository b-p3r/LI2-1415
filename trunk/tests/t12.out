A
ACAIME
ACARO
ACAROS
ACATA
ACATASSE
ACATASSEM
ACATE
ACATEM
ACATO
ACCOES
ACEITO
ACESO
ACESSO
ACESSORIO
ACESSOS
ACIMA
ACLARE
ACNE
ACO
ACOITA
ACOITAR
ACOITES
ACOS
ACOSSO
ACUDE
ACUSO
ADAGA
ADAGAS
ADE
ADMIRA
ADMIRADA
ADMIRADAS
ADMIRADO
ADMIRADOR
ADMIRADORA
ADMIRADOS
ADMIRAR
ADMIRARA
ADMIRARAO
ADMIRE
ADMIRO
ADMIROU
ADO
ADOCA
ADORA
ADORADA
ADORADAS
ADORADO
ADORADOR
ADORADOS
ADORAR
ADORARA
ADORAREM
ADORARIA
ADORE
ADOREI
ADOREM
ADORO
ADOROU
ADRO
ADULAR
ADVIR
ADVOGA
ADVOGAM
ADVOGAMOS
ADVOGASSE
ADVOGO
AEREO
AEREOS
AFA
AFOITA
AFOITO
AFOITOU
AFORA
AFORRADO
AFORRO
AFRO
AGATA
AGATAS
AGOIRO
AGOIROS
AGUA
AGUADA
AGUADAS
AGUADO
AGUADOS
AGUAR
AGUARDA
AGUARDAM
AGUARDE
AGUARDEI
AGUARDEM
AGUARDO
AGUAS
AGUDA
AGUDAS
AGUDO
AGUDOS
AI
AIA
AIAS
AIO
AIRES
AIS
ALA
ALADO
ALADOS
ALAGA
ALAGAM
ALAR
ALARDE
ALARDEIA
ALARDEIAM
ALARMA
ALARMADO
ALARMAM
ALARME
ALARMES
ALARMOU
ALAS
ALCA
ALCANCA
ALCANCASSE
ALCANCASSEM
ALCANCO
ALCAR
ALCAS
ALCATRA
ALCATRAO
ALCATRAS
ALDEIA
ALDEIAS
ALI
ALIA
ALIAM
ALIAS
ALIE
ALIEM
ALISA
ALISADO
ALISAM
ALO
ALUADO
ALUADOS
ALUDA
ALUDO
ALUGA
ALUGADO
ALUGADOS
ALUGAM
ALUGAMOS
ALUGASSE
ALUGO
ALUSAO
AMA
AMAGO
AMAIS
AMAS
AMASSE
AME
AMEI
AMEIA
AMEIAS
AMEM
AMEN
AMENA
AMERICA
AMERICAS
AMIANTO
AMIUDE
AMO
AMOLA
AMOLADOR
AMOLAR
AMOR
AMORA
AMORAL
AMOREIRA
AMORES
AMORFA
AMORFO
AMORFOS
AMOS
AMOU
ANA
ANAL
ANCA
ANCAS
ANCIA
ANCIAO
ANCIAOS
ANCIAS
ANCIOES
ANEMIA
ANEMIAS
ANEMICA
ANEMICAS
ANEMICO
ANEURISMA
ANIMA
ANIMAL
ANIME
ANIMICA
ANIMICAS
ANIMICO
ANIMISMO
ANIMISTA
ANO
ANOES
ANTA
ANTAO
ANTAS
ANTE
ANTES
ANTI
ANTRO
ANTROS
AO
AORTA
AOS
AR
ARA
ARADA
ARADO
ARADOS
ARAR
ARARA
ARAS
ARDA
ARDAM
ARDE
ARDEM
ARDOR
ARE
AREA
AREADO
AREAS
AREIA
ARES
ARFA
ARFAR
ARIA
ARMA
ARMADO
ARMADOR
ARMADORA
ARMAM
ARMAS
ARME
ARMEIRO
ARMES
ARMOU
ARO
AROMA
AROMAS
AROS
ARRE
ARREADO
ARREIA
ARROLA
ARROLADA
ARROLADAS
ARROLADO
ARROLADOS
ARROTA
ARROTO
ARROTOS
ARTE
ARTES
ARTESA
ARTESAO
ARTESAOS
AS
ASA
ASCO
ASIA
ASILADO
ASILADOS
ASILAR
ASILO
ASMA
ASNO
ASO
ASSEIO
ASSIM
ASSOMO
ATA
ATACA
ATACAR
ATACAREM
ATACAREMOS
ATACO
ATAR
ATARA
ATAREM
ATE
ATICA
ATINA
ATINAR
ATINAREM
ATO
ATOIS
ATOMISMO
ATOMO
ATOMOS
ATOU
ATRACA
ATRAI
ATRAIA
ATRAICOA
ATRAICOAM
ATRAS
ATRIO
ATRIOS
AULA
AULAS
CA
CACE
CACEM
CACO
CACOS
CAGA
CAGADA
CAGADO
CAI
CAIA
CAIACAO
CAIAR
CAIAS
CAIEM
CAIO
CAIS
CAL
CALA
CALAI
CALAM
CALAMITOSA
CALAMITOSO
CALAO
CALAR
CALAREI
CALAREM
CALAREMOS
CALARES
CALAS
CAMA
CAMAS
CAMELO
CANA
CANAL
CANCAO
CANICO
CANO
CANOEIRO
CANORA
CANORO
CANOROS
CANTA
CANTAI
CANTAO
CANTAR
CANTAREI
CANTAREM
CANTAREMOS
CANTARES
CANTARO
CANTAROS
CANTAS
CANTASSE
CANTASSEM
CANTASSEMOS
CANTE
CANTEI
CANTEIRO
CANTEIROS
CANTEM
CANTEMOS
CANTICO
CANTO
CANTOES
CANTOR
CANTORA
CANTORAS
CANTORES
CANTOS
CAO
CAOS
CARA
CARAO
CARAS
CARETA
CARETAS
CARETO
CARETOS
CARIE
CARIES
CARNAL
CARO
CAROS
CARTA
CARTAO
CARTAS
CARTE
CARTES
CARTOES
CASA
CASADA
CASADO
CASADOIRO
CASADOIROS
CASADOS
CASAL
CASAM
CASAMOS
CASAO
CASCO
CASE
CASEI
CASEIRA
CASEIRAS
CASEIRO
CASEM
CASERNA
CASERNAS
CASO
CASORIO
CASOS
CASOTA
CASOTAS
CASUAL
CASULO
CATA
CATALA
CATALAO
CATALAS
CATANA
CATAR
CATRAIA
CATRAIAS
CATRAIO
CE
CEGO
CEIA
CEIAS
CEM
CESIO
CESSA
CESSAM
CESSAO
CESSE
CESSEI
CESSEM
CESSO
CESTO
CETIM
CILADA
CILADAS
CIMA
CIMENTA
CIMENTACAO
CIMENTAR
CIMENTARA
CIMENTAREM
CIMENTE
CIMENTEIRA
CIMENTEIRAS
CIMENTEM
CIMENTO
CIMENTOS
CIMO
CINEMA
CINTA
CINTAS
CINTO
CINTOS
CIO
CIOSA
CIOSO
CIOSOS
CISAO
CISMA
CISMOU
CITA
CITACAO
CITAR
CITARA
CITARAO
CITARAS
CITAREI
CITAREM
CITAREMOS
CITAS
CITASSE
CITE
CITEI
CITEM
CITEMOS
CITES
CITO
CIUME
CIUMENTA
CIUMENTAS
CIUMENTO
CIUMENTOS
CLA
CLAME
CLARA
CLARAO
CLARAS
CLARIM
CLARO
CLAROES
CLAROS
CLAS
COA
COADA
COADO
COADOR
COAGULA
COAGULADO
COAGULADOS
COAGULAR
COAGULO
COESO
COIMA
COIOTE
COIOTES
COIRO
COIROS
COISA
COISO
COITA
COITO
COITOS
COLE
COLEM
COM
COMA
COMAS
COME
COMES
COMESSE
COMESSEM
COMETI
COMETO
COSA
COSE
COSEM
COSEMOS
COSER
COSI
COSIA
COSMETICA
COSMETICO
COSMICA
COSMICO
COSMOS
COSO
COTE
COVIS
COVO
CU
CUME
CURE
CUREM
CURIOSO
CURO
CURSO
CUS
CUSTA
CUSTAR
CUSTARA
CUSTARAO
CUSTARIA
CUSTO
CUSTOU
DA
DALI
DALIAS
DAME
DAO
DAR
DARA
DARAO
DARDO
DAREI
DAREM
DARES
DARIA
DAS
DE
DEEM
DEI
DEISMO
DEMITA
DEMITE
DEMITEM
DEMITO
DEMO
DEMOLIA
DEMOLIAM
DEMOLICAO
DEMORA
DEMORADA
DEMORADAS
DEMORADO
DEMORADOS
DEMORAR
DEMORARA
DEMORARAO
DEMORAREM
DEMORARIA
DEMORE
DEMOREI
DEMOREM
DEMORO
DEMOROU
DENTAL
DENTARIO
DENTARIOS
DENTE
DENTES
DENTICAO
DENTO
DENTRE
DENTRO
DER
DEREM
DERIVOU
DERMOS
DEU
DEUS
DO
DOA
DOADA
DOADAS
DOADO
DOADOR
DOADORA
DOADOS
DOAM
DOAR
DOARA
DOAREM
DOARIA
DOAS
DOASSE
DOCA
DOCAS
DOCUMENTA
DOCUMENTACAO
DOCUMENTAL
DOCUMENTAR
DOCUMENTAREM
DOCUMENTARIO
DOCUMENTARIOS
DOCUMENTE
DOCUMENTO
DOCUMENTOS
DOI
DOIA
DOIS
DOLAR
DOLARES
DOM
DOME
DOMO
DOO
DOOU
DOR
DORES
DORME
DORMES
DORMI
DORMIA
DORMIR
DORMIRA
DORMIRAO
DOS
DOTA
DOTAR
DOTARA
DOTARAO
DOTARIA
DOTOU
DOU
DRUSO
DUAL
DUALISMO
DUAS
DUM
DUMA
DUO
DURE
DUREM
DURMA
DURMAM
DURMO
DURO
DUROS
E
ECO
ECOA
ECOADA
ECOADO
ECOAM
ECOASSE
ECOASSEM
ECOE
ECOEM
ECOS
EDEMA
EDEN
EDUCA
EGO
EGOISMO
EIA
EIRA
EIRAS
EIS
EITO
ELE
ELEGE
ELEITO
ELES
ELO
EM
EMAIL
EMANA
EMANACAO
EMANAR
EMANAREM
EMIR
EMIRADO
EMISSAO
EMISSOR
EMISSORA
EMISSORAS
EMITA
EMITE
EMITEM
EMITO
ENA
ENCIMA
ENLACA
ENLACAR
ENLATAR
ENTALA
ENTALAM
ENTALAR
ENTAO
ENTE
ENTES
ENTOA
ENTOAM
ENTOAMOS
ENTOASSE
ENTOASSEM
ENTORNA
ENTORNO
ENTRA
ENTRAI
ENTRAS
ENTRE
ENTREI
ENTREM
ENTREMOS
ENTRES
ENTRO
ENTRONCA
ERA
ERAS
ERMITA
ERMITAO
ERMO
ERMOS
EROSAO
EROSIVO
EROTICA
EROTICAS
EROTICO
ERRA
ERRADA
ERRADAS
ERRADO
ERRADOS
ERRO
ERROU
ES
ESCAMA
ESCAMAS
ESCAMOSA
ESCOA
ESCOAM
ESCOE
ESMAGA
ESMAGADA
ESMAGADO
ESMAGADOR
ESMAGADORA
ESMAGADOS
ESMERA
ESMERO
ESMOLA
ESMOLAS
ESSA
ESSE
ESTIME
ESTIMEI
ESTIMO
ESTIO
ETANO
ETARIO
ETARIOS
ETICA
ETICAS
ETICO
ETIMO
ETNIA
ETNIAS
ETNICA
ETNICAS
ETNICO
EU
EURO
EUROS
FA
FADA
FADAS
FADO
FADOS
FALA
FALADO
FALADOR
FALADORA
FALADOS
FALAS
FALICA
FALICAS
FALICO
FALO
FALOU
FALSA
FALSO
FALSOS
FALUAS
FARA
FARAO
FARDA
FARDAS
FARDO
FAREI
FARIA
FARO
FAROL
FAROLIM
FARRA
FATIA
FATO
FATOS
FE
FEL
FLOCO
FLOCOS
FOCA
FOCAIS
FOCAM
FOCAS
FOCO
FOCOS
FOGE
FOGO
FOI
FOLE
FOLEGO
FOLES
FOMOS
FOR
FORA
FORAL
FORO
FORRA
FORRADA
FORRADAS
FORRADO
FORRADOS
FORRE
FORRO
FORROU
FOTAO
FOTO
FOTOS
FRALDA
FRALDAS
FRIA
FROTA
GADO
GADOS
GAIA
GAIATA
GAIATO
GAIATOS
GAIO
GAITA
GAITAS
GAITEIRA
GAITEIRO
GAITEIROS
GAL
GALA
GALAO
GALAR
GALARIM
GALIO
GALO
GAMA
GAMAO
GAMAS
GAME
GAMES
GAMO
GAMOS
GAS
GASOMETRO
GASOSA
GASOSO
GATA
GATAS
GATES
GATO
GATOS
GEL
GELES
GELO
GODO
GODOS
GOES
GOLE
GOLES
GOLFE
GOMA
GOMES
GUARDA
GUARDAM
GUARDAS
GUARDE
GUARDEI
GUARDEM
GUARDO
GULA
GULAS
IA
IAM
IAMOS
IAO
IAS
IATE
IATES
ICA
ICAM
ICE
ICEM
ILUDA
ILUDO
ILUSAO
IMA
IMAN
IMITA
IMITACAO
IMITAR
IMITARA
IMITARAO
IMITAREM
IMITASSE
IMITASSEM
IMITE
IMITEM
IMITO
IMOLA
IMOLADA
IMOLADAS
IMOLADO
IMOLADOS
IMOLAR
IMORAL
IMORTAL
INALA
INALACAO
INALAM
INALAR
INATA
INATAS
INATO
INATOS
INCA
INCAS
INCISO
INTEIRA
INTEIRAS
INTEIRO
INTEIROS
INTERNA
INTERNAS
INTERNO
INTRA
INTROMISSAO
IODO
IOES
IOGA
IOIO
IR
IRA
IRADA
IRADAS
IRADO
IRADOS
IRAO
IRAS
IRE
IREM
IREMOS
IRES
IRMA
IRMAO
IRMAOS
IRMAS
IRMOS
IRRA
IS
ISCA
ISCAS
ISCO
ISCOS
ISLAO
ISOMERO
ISSO
ISTMO
ISTO
ITEM
ITERACAO
LA
LACA
LACAIO
LACAIOS
LACAR
LACAROTE
LACAROTES
LACAS
LADO
LADOS
LAGO
LAGOA
LAGOAS
LAGOS
LAIA
LAICA
LAICAS
LAICISMO
LAICO
LAMENTA
LAMENTACAO
LAMENTAR
LAMENTARA
LAMENTARAO
LAMENTAREI
LAMENTAREM
LAMENTAREMOS
LAMENTASSE
LAMENTE
LAMENTEI
LAMENTEM
LAMENTEMOS
LAMENTO
LAMENTOS
LAMENTOSO
LAMINA
LAMINACAO
LAMINAR
LAMIRE
LANCA
LANCAR
LANCAREI
LANCAREM
LANCAREMOS
LANCAS
LANCASSE
LANCASSEM
LANCO
LANTERNA
LANTERNAS
LAR
LAREIRA
LARES
LAS
LASCA
LASCAR
LATA
LATAO
LATAS
LATE
LATIM
LATINA
LATO
LATOEIRO
LATOES
LE
LEGO
LEI
LEIO
LEIS
LEITO
LEMA
LEMAS
LEMOS
LEOES
LES
LESMA
LESMAS
LESOES
LESSE
LESSEM
LESTO
LI
LIA
LIAM
LIAMOS
LICA
LICAO
LIMO
LIMOU
LISA
LISO
LISOS
LO
LOCA
LOCAIS
LOCAO
LOCO
LOGO
LORD
LORDE
LOTE
LOTES
LOUSA
LUA
LUAR
LUARES
LUAS
LUSA
LUSO
LUSOS
MA
MACE
MACEM
MACIO
MACO
MACOS
MADRE
MADRES
MAE
MAES
MAGA
MAGAS
MAGO
MAGOA
MAGOADA
MAGOADO
MAGOADOS
MAGOAS
MAGOASSE
MAGOS
MAIO
MAIOR
MAIS
MAL
MALA
MALAIA
MALAIAS
MALAIO
MALAIOS
MALAR
MALARES
MALAS
MAMOU
MANA
MANCA
MANCO
MANE
MANIA
MANIACA
MANIACAS
MANIACO
MANIAS
MANTA
MANTAS
MANTEM
MANTEMOS
MANTER
MANTERA
MANTERAO
MANTO
MANTOS
MAO
MAOISTA
MAOMETANO
MAOS
MAS
MASMORRA
ME
MEADO
MECA
MECAM
MECAS
MECO
MEDA
MEDAS
MEDE
MEDO
MEDRA
MEDRAR
MEDRICAS
MEDROSO
MEDROU
MEIA
MEIAS
MEIO
MEIOS
MEL
MELO
MEMORIA
MEMORIES
MENCAO
MENTA
MENTAL
MENTE
MENTEM
MENTES
MENTI
MENTIA
MENTOR
MENTORA
MENTORAS
MENTORES
MERA
MERAS
MERDA
MERDAS
MERO
MEROS
MES
MESA
MESMA
MESMAS
MESMO
MESMOS
MESSE
META
METAL
METANO
METAS
METI
METIA
METO
METRO
METROS
MEU
MEUS
MI
MIA
MIAM
MIAR
MIAS
MICA
MIL
MINA
MINAR
MINARA
MINAREM
MINE
MINTA
MINTO
MIO
MIRA
MIRADA
MIRADAS
MIRADO
MIRADOIRO
MIRANTE
MIRANTES
MIRAR
MIRAS
MIRE
MIROU
MIRRA
MIRRADA
MIRRADAS
MIRRADO
MIRRADOS
MISSA
MISSAO
MISSE
MISTA
MISTO
MITO
MITOS
MITOSE
MITRA
MITRAL
MITRAS
MO
MOA
MOAS
MOCA
MOCAS
MOCO
MOCOES
MODA
MODAL
MODALISMO
MODAS
MODEM
MODEMS
MODO
MODORRA
MODULA
MODULAR
MODULARES
MODULO
MOEM
MOER
MOEU
MOFA
MOFO
MOI
MOIAM
MOINA
MOIRA
MOIRAS
MOIRO
MOIROS
MOITA
MOITAS
MOLA
MOLAR
MOLARES
MOLAS
MOLDA
MOLDAM
MOLDE
MOLICO
MOLUSCO
MOR
MORA
MORADA
MORADAS
MORADO
MORADOR
MORADORA
MORAL
MORAR
MORARA
MORAREM
MORAS
MORDA
MORDAM
MORDE
MORDEM
MORDEU
MORE
MOREI
MOREIA
MOREM
MORES
MORNA
MORNAS
MORNO
MORO
MOROU
MORRA
MORRE
MORREM
MORRES
MORRI
MORRIA
MORRO
MORTA
MORTAL
MORTAS
MORTE
MORTES
MORTICA
MORTICAS
MORTICO
MOS
MOSAICO
MOSCA
MOSCAS
MOSSA
MOSSO
MOSTO
MOTA
MOTAS
MOTE
MOTES
MOTIM
MOTO
MOTOR
MOTORA
MOTOS
MOVI
MOVIMENTA
MOVIMENTACAO
MOVIMENTAR
MOVIMENTARA
MOVIMENTARAO
MOVIMENTAREM
MOVIMENTASSE
MOVIMENTASSEM
MOVIMENTE
MOVIMENTEI
MOVIMENTEM
MOVIMENTO
MOVIMENTOS
MOVO
MUCO
MUDE
MUI
MURO
MUROS
MUSICA
MUSICAS
MUSICO
NA
NACAO
NAO
NAS
NASCA
NATA
NATAL
NATAS
NATO
NATOS
NEM
NICAS
NICO
NITROSO
NO
NORA
NORAS
NORTE
O
OCA
OCAS
OCASO
OCIO
OCIOS
OCIOSA
OCIOSO
OCIOSOS
OCO
OCOS
ODALISCA
ODE
ODEIA
ODEIAM
ODEIE
ODEIEM
ODEIO
ODOR
ODRE
ODRES
OE
OICA
OICAM
OICO
OIRO
OIROS
OITO
OLA
OLARIA
OLE
OMISSA
OMISSAO
OMISSO
OMISSOS
OMITA
OMITE
OMITEM
OMITO
ONCA
ONCAS
ORA
ORACAO
ORADOR
ORADORA
ORAI
ORAL
ORANTE
ORANTES
ORAR
ORAREM
ORAS
ORDEM
ORDENA
ORDENACAO
ORDENAM
ORDENAR
ORDENARA
ORDENARAO
ORDENAREM
ORE
OREMOS
ORFA
ORFAO
ORNA
ORNATO
ORNATOS
ORO
OROU
OS
OSCILA
OSCILADO
OSCILAR
OSCILARA
OSCILARAO
OSCILAREM
OSCILARIA
OSCILO
OSCILOU
OSSEO
OSSO
OSSOS
OU
OUSA
OUSADA
OUSADO
OUSADOS
OUSAM
OUSAMOS
OUSO
OVO
OVOCITO
OVOCITOS
OVOS
RA
RACA
RACAO
RACAS
RAIA
RAIAS
RAIO
RAIOS
RALA
RALADO
RALAM
RALI
RALIS
RALO
RALOU
RANCO
RARA
RAREADO
RAREIA
RARO
RAS
RASCA
RATA
RATAO
RATAS
RATE
RATEIO
RATES
RATO
RATOS
RE
REDE
REDEA
REDENCAO
REDENTOR
REDENTORA
REDENTORAS
REDENTORES
REDUCAO
REI
REMA
REMADO
REMADOR
REMADORA
REMAM
REMIR
REMISSAO
REMO
REMOEU
REMOI
REMOS
REMOTA
REMOTAS
REMOU
RES
RESMA
RESMAS
RETAGUARDA
RETINA
RETOMO
REU
REUS
RI
RIA
RICA
RICAS
RICO
RIEM
RIM
RIMA
RIMADO
RIMAM
RIMAS
RIMO
RIMOS
RIMOU
RIO
RIOS
RIR
RIRA
RIRAO
RIREM
RIRES
RIS
RISO
RISOTA
RIU
RO
ROA
ROAM
RODA
RODAM
RODAR
RODARA
RODAREM
RODARIA
RODAS
RODE
RODEI
RODEIA
RODEIAM
RODEIE
RODEIEM
RODEIO
RODEM
RODO
RODOS
ROEM
ROEU
ROI
ROIA
ROIAM
ROL
ROLA
ROLADAS
ROLADO
ROLADOS
ROLAO
ROLAR
ROLARA
ROLARAO
ROLAREM
ROLARIA
ROLAS
ROLICA
ROLICAS
ROLICO
ROMA
ROMAS
ROMEIRO
RONCA
ROSA
ROSE
ROSEO
ROSEOS
ROSETAS
ROSSIO
ROSTO
ROTA
ROTACAO
ROTAS
ROTINA
ROTO
ROTOS
RUCA
RUCAS
RUCO
RUDE
RUI
RUIM
RUIVO
RUIVOS
RUM
RUMA
RUME
RUMINA
RUMINACAO
RUMINAR
SA
SACA
SACANA
SACAR
SACAREM
SACAROSE
SACIA
SACIAR
SACIAREM
SACO
SACOS
SADO
SAEM
SAGA
SAGAS
SAI
SAIA
SAINT
SAIO
SAIOTE
SAL
SALA
SALAME
SALAO
SALARIO
SALARIOS
SALDA
SALDAM
SALDE
SALDEM
SALDO
SALDOU
SALMO
SAM
SAN
SANO
SAO
SAOS
SARA
SARE
SAREM
SATA
SATANICA
SATANICAS
SATANICO
SAUDA
SAUDAR
SAUDARA
SAUDARAO
SAUDAREM
SAUDARIA
SAUDO
SAUDOSO
SCALA
SE
SECA
SECAM
SECAS
SECCAO
SECO
SECOS
SEI
SEIO
SEIOS
SELE
SELO
SEM
SEMOLA
SER
SERA
SERAO
SERAS
SERIA
SERIAR
SERIO
SERIOS
SERRA
SERRADA
SERRADAS
SERRADO
SERRADOS
SERRIM
SERROU
SERTA
SERTAO
SETA
SETAS
SETIMA
SETIMO
SETIMOS
SI
SIAMES
SILO
SIM
SIMIO
SIMIOS
SIMOES
SIR
SITE
SITO
SO
SOA
SOADO
SOAM
SOASSE
SOASSEM
SOCA
SOCAIS
SOCAS
SOCO
SODA
SOE
SOEM
SOFA
SOFRA
SOFRI
SOFRIA
SOFRO
SOM
SOMA
SOMAM
SOMAS
SOMASSE
SOME
SOMEI
SOMO
SOMOS
SOOU
SOR
SORNA
SORO
SORTE
SORTEIO
SORTEIOS
SOS
SOSIA
SOTAINA
SOTAO
SOVOU
SUA
SUADA
SUADO
SUADOS
SUAO
SUAR
SUAREM
SUCATA
SUCATAS
SUCATEIRO
SUCATEIROS
SUCINTA
SUCINTAS
SUCINTO
SUCINTOS
SUCO
SUE
SUGA
SUGADA
SUGADO
SUGADOR
SUGADORA
SUGADOS
SUGAM
SUICA
SUICAS
SUICO
SUL
SUMA
SUMIA
SUMICO
SUMIR
SUOR
SUORES
TA
TACA
TACAO
TACAS
TACO
TAL
TALA
TALAO
TALAS
TANOEIRO
TANOEIROS
TAO
TAOISTA
TARA
TARADA
TARADAS
TARADO
TARADOS
TARAS
TARO
TASCA
TE
TECA
TECAM
TECE
TECO
TEIME
TEIMEI
TEIMO
TEIMOSA
TEIMOSO
TEIMOSOS
TEISMO
TELE
TEM
TEMA
TEMAIS
TEMAS
TEMI
TEMO
TEMOR
TEMOS
TEOR
TER
TERA
TERAO
TERAS
TERNA
TERNAS
TERNO
TESA
TESAO
TESO
TESOIRO
TESOS
TI
TIA
TIARA
TIAS
TICAO
TIM
TIMO
TIMOR
TINA
TIO
TIOS
TO
TOA
TOADA
TOADAS
TOCA
TOCAM
TOCAMOS
TOCAS
TOCO
TODA
TODAS
TODO
TODOS
TOIRO
TOIROS
TOM
TOMA
TOMAM
TOMAMOS
TOME
TOMEI
TOMO
TOMOS
TOO
TORA
TORIO
TORNA
TORNAS
TORNO
TORO
TORRA
TORRADA
TORRADAS
TORRADO
TORRADOS
TORRAO
TORRE
TORRES
TOSSE
TOSSEM
TRACA
TRACAS
TRAI
TRAIA
TRAICAO
TRANCA
TRANCAS
TRAS
TREM
TREMI
TREMO
TRES
TRIO
TRIOS
TROIANA
UI
UIVO
UIVOS
UIVOU
UM
UMA
URDE
URICO
URSO
USA
USADA
USADO
USADOS
USAM
USAMOS
USO
USOS
VICIA
VICIACAO
VICIANTE
VICIANTES
VICIAR
VICIARA
VICIAREM
VICIASSE
VICIO
VICIOS
VICIOSA
VICIOSO
VICIOSOS
VIM
VIME
VIR
VIRE
VIREM
VIREMOS
VIRMOS
VIRO
VIRUS
VIS
VISO
VISOR
VISTA
VISTO
VISTORIA
VISTORIAR
VISTORIAREM
VIU
VOA
VOADAS
VOADO
VOADOS
VOAM
VOAMOS
VOAR
VOARA
VOARAO
VOAREM
VOARES
VOARIA
VOASSE
VOASSEM
VOGA
VOGADO
VOGAL
VOGAM
VOMITA
VOMITAR
VOMITO
VOMITOU
VOO
VOOS
VOOU
VOS
VOTA
VOTAR
VOTARA
VOTARAO
VOTARIA
VOTO
VOTOS
VOTOU
VOU
